azure-functions/acbs-function/src/functions/acbs-amend-facility-loan-record.js

```diff
@@ -32,57 +32,57 @@ df.app.orchestration('acbs-amend-facility-loan-record', function* amendFacilityL
   const payload = context.df.input;
 
   try {
-    if (payload) {
-      const { facilityId, facility, amendments, fmr } = payload;
-      const { amendment } = amendments;
-      let facilityLoanRecordAmendments;
+    if (!payload) {
+      throw new Error('Invalid payload provided');
```

Comment:
Should we be more specific here, i.e if this is logged out will we be able to tell where the error was thrown? Otherwise i'm thinking could be e.g

```js
throw new Error('Facility Loan Record amendment SOF - Invalid payload provided');
```

Reply:
good idea

Reply:
Good shout, @AlexBramhill  might be prudent to refactor other `Invalid payload provided` generic error message I wrote happy to be a different PR. Thanks